wandb/sdk/launch/launch.py

```diff
@@ -277,18 +330,3 @@ def run(
     )
 
     return submitted_run_obj
-
-
-def _wait_for(submitted_run_obj: AbstractRun) -> None:
```

Comment:
Was this just unused?

Reply:
yes, the wait is just implemented in each runner now. should we keep this around?

Reply:
No this si fine